Management_System/BLL/Services/Implementaiton/TaskStatusService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using BLL.Services.Interfaсes;
+using DAL.Entities;
+using DAL.Repositories.Interfaces;
+
+namespace BLL.Services.Implementaiton;
+
+public class TaskStatusService(ITaskStatusRepository repository) : ITaskStatusService
+{
+    public async Task<TaskStatusEntity> Add(TaskStatusEntity entity, CancellationToken cancellationToken)
+    {
+        await repository.Add(entity, cancellationToken);
+
+        return entity;
+    }
```

Comment:
And here we back to my previous question.
Will be `entity` updated after you have added it to a db?
 e.g. entity's id

Reply:
![image](https://github.com/user-attachments/assets/ee277449-6eb8-4790-9507-c3ca141ec7b8)
here is the result of executing add method

Reply:
I use AddAsync method, that tracking the given entity, and any other reachable entities that are not already being tracked